cli/app/runtime_attachment.go

```
@@ -115,6 +117,22 @@ func prepareSharedRuntimeWiring(ctx context.Context, clients runtimeAttachmentCl
 	}, runtimeClient.transcriptDiagnosticsEnabled, func(line string) {
 		logger.Logf("%s", line)
 	})
+	var transcriptEvents <-chan ongoingTranscriptEvent
+	requestTranscriptOpen := func() {}
```

golangci-lint (2.12.2)

[error] ineffectual assignment to requestTranscriptOpen

(ineffassign)